hw/top_earlgrey/dv/vendor_chip_sim_cfg_sample.hjson

```diff
@@ -0,0 +1,15 @@
+// Copyright lowRISC contributors.
```

Comment:
@eunchan 
THis is an example of how Nuvoton can simply import our chip sim cfg hjson and just use the overrides directive to modify the design / tb.

Reply:
Thanks!